eng/common/TestResources/TestResources-Helpers.ps1

```diff
@@ -203,19 +203,34 @@ function SetDeploymentOutputs(
     $deploymentOutputs = BuildDeploymentOutputs $serviceName $azContext $deployment $deploymentEnvironmentVariables
 
     if ($OutFile) {
-        if (!$IsWindows) {
-            Write-Host 'File option is supported only on Windows'
-        }
+        if ($IsWindows -and $isDotnet) {
```

Comment:
Is the $isDotNet variable in scope here?

Reply:
Yes. Variable are in scope for the current and child scopes. Since we source this file, it's in the same scpe as the rest of the script.